versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5846 @@
+  }
+}
+
+workflow foo {
+  input {
+    String name
+    Boolean use_salutation
+  }
+  
+  if (use_salutation) {
+    call say_hello { input: name = name }
+  }
+
+  if (!use_salutation) {
+    call say_hello { 
+      input: 
+        name = name,
+        salutation = None 
+    }
+  }
+}
+```
+
+### Private Declarations
+
+A task can have declarations that are intended as intermediate values rather than inputs. These private declarations may appear anywhere in the body of the task, and they must be initialized. Just like input declarations, private declarations may be initialized with literal values, or with expressions that may reference other declarations.
+
+For example, this task takes an input and then performs a calculation, using a private declaration, that can then be referenced in the command template:
+
+```wdl
+task t {
+  input {
+    Int size
+  }
+
+  Int size_clamped = if size > 10 then 10 else size
+  ...
+}
+```
+
+The value of a private declaration may *not* be specified by the task caller, nor is it accessible outside of the current scope.
+
+```wdl
+task test {
+  input {
+    Int i
+  }
+  String s = "hello"
+  command { ... }
+  output {
+    File out = "/path/to/file"
+  }
+}
+
+workflow wf {
+  call test {
+    input:
+      i = 1,         # this is fine - "i" is in the input section
+      s = "goodbye"  # error! "s" is private
+  }
+  output {
+    File out = test.out # this is fine - "out" is in the output section
+    String s = test.s   # error! "s" is private
+  }
+}
+```
+
+### Command Section
+
+The `command` section is the only required task section. It defines the command template that is evaluated and executed when the task is called. Specifically, the commands are executed after all of the inputs are staged and before the outputs are evaluated.
+
+There are two different syntaxes that can be used to define the command section:
+
+```wdl
+# HEREDOC style - this way is preferred
+command <<< ... >>>
+
+# older style - may be preferable in some cases
+command { ... }
+```
+
+There may be any number of commands within a command section. Commands are not modified by the execution engine, with the following exceptions:
+- Common leading whitespace is [stripped](#stripping-leading-whitespace)
+- String interpolation is performed on the entire command section to replace [expression placeholders](#expression-placeholders) with their actual values.
+
+#### Expression Placeholders
+
+The body of the command section (i.e. the command "template") can be though of as a single string expression, which (like all string expressions) may contain placeholders.
+
+There are two different syntaxes that can be used to define command expression placeholders, depending on which style of command section definition is used:
+
+|Command Definition Style|Placeholder Style|
+|---|---|
+|`command { ... }`|`~{}` (preferred) or `${}`|
+|`command <<< >>>`|`~{}` only|
```

Comment:
minutia:  put the <<< >>> option first so it looks more like it's the preferred option for the format.  Then maybe put a caveat for the  {  } format to remind folks that when that is used you might run into bash variable problems.

Reply:
done